platform/broadcom/sonic-platform-modules-cel/silverstone2/modules/i2c-ocores.c

```diff
@@ -0,0 +1,846 @@
+// SPDX-License-Identifier: GPL-2.0
+/*
+ * i2c-ocores.c: I2C bus driver for OpenCores I2C controller
+ * (https://opencores.org/project/i2c/overview)
+ *
+ * Peter Korsgaard <peter@korsgaard.com>
+ *
+ * Support for the GRLIB port of the controller by
+ * Andreas Larsson <andreas@gaisler.com>
+ */
+
+#include <linux/clk.h>
```

Comment:
use the linux kernel one?

Reply:
I will update this PR to use kernel's i2c-ocores after this https://github.com/Azure/sonic-linux-kernel/pull/135 merged.

Reply:
pr has been merged.